src/SignhostAPIClient/Rest/DataObjects/Verification.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Signhost.APIClient.Rest.DataObjects
+{
+	public class Verification
```

Comment:
class should not be instantiated, maybe abstract?
otoh this could give problems when deserializing an unknown verification object?